payments-core/src/main/java/com/stripe/android/view/MaskedCardView.kt

```diff
@@ -84,6 +84,8 @@ internal class MaskedCardView @JvmOverloads constructor(
                     CardBrand.Visa -> R.drawable.stripe_ic_visa_template_32
                     CardBrand.MasterCard -> R.drawable.stripe_ic_mastercard_template_32
                     CardBrand.UnionPay -> R.drawable.stripe_ic_unionpay_template_32
+                    // TODO: (jameswoo) get cartesbancaires asset
```

Comment:
I pinged #payments-design here:
https://stripe.slack.com/archives/C0159J1LYSC/p1674094653565129

You should be able to export svgs here:
https://www.figma.com/file/RhAvXp2cVFxFInrPINEhMo/%E2%9B%B5Checkout-Design-System?node-id=14%3A334&t=Q7jbV41Nw4ELJszm-0